node/insecure_validator_manager.go

```diff
@@ -4,38 +4,53 @@
 package node
 
 import (
+	"go.uber.org/zap"
+
 	"github.com/ava-labs/avalanchego/ids"
 	"github.com/ava-labs/avalanchego/snow/networking/router"
 	"github.com/ava-labs/avalanchego/snow/validators"
-	"github.com/ava-labs/avalanchego/utils/constants"
+	"github.com/ava-labs/avalanchego/utils/logging"
 	"github.com/ava-labs/avalanchego/version"
 )
 
 type insecureValidatorManager struct {
 	router.Router
-	vdrs   validators.Set
-	weight uint64
+	log      logging.Logger
+	subnetID ids.ID
```

Comment:
Why do we generalize the subnetID here? We used the `PrimaryNetworkID` here previously because we were guaranteed that the networking code would call the router with that subnetID (regardless of the `TrackedSubnets` flag)

Reply:
removed